rxandroid/src/main/java/io/reactivex/android/schedulers/AndroidSchedulers.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.reactivex.android.schedulers;
+
+import android.os.Handler;
+import android.os.Looper;
+import io.reactivex.Scheduler;
+import io.reactivex.android.plugins.RxAndroidPlugins;
+
+/** Android-specific Schedulers. */
+public final class AndroidSchedulers {
+    private static final Scheduler MAIN_THREAD = RxAndroidPlugins.initMainThreadScheduler(
+        new HandlerScheduler(new Handler(Looper.getMainLooper())));
+
+    /** A {@link Scheduler} which executes actions on the Android UI thread. */
```

Comment:
nit: it's not specifically UI thread, it's just main thread of the Android app process (it can be UI-less)